app_dart/test/src/service/fake_github_service.dart

```diff
@@ -62,8 +64,19 @@ class FakeGithubService implements GithubService {
   }
 
   @override
-  Future<String> getFileContent(RepositorySlug slug, String path) async {
-    return '';
+  Future<String> getFileContent(RepositorySlug slug, String path, {String? ref}) async {
+    if (path == 'bin/internal/release-candidate-branch.version') {
+      String encodedResponse = '';
+      if (ref == 'beta') {
+        encodedResponse = 'Zmx1dHRlci0zLjItY2FuZGlkYXRlLjUK\n';
```

Comment:
1. Avoid making test values unreadable to humans.

2. Do we need to handle encoding in the fake? It seems like its encoded then decoded

```dart
final String encodedResponse = base64.encode(utf8.encode('flutter-3.2-candidate.5\n'));
```

Or simpler:

```dart
// Adding async to the method doesn't inherently makes all returned values a Future
if (ref == 'beta') {
  return 'flutter-3.2-candidate.5\n';
}
```